sdk/internal/env/env.go

```diff
@@ -41,31 +41,61 @@ const (
 	// i.e. 512
 	BatchSpanProcessorMaxExportBatchSizeKey = "OTEL_BSP_MAX_EXPORT_BATCH_SIZE"
 
-	// SpanAttributeValueLengthKey
+	// AttributeValueLengthKey
 	// Maximum allowed attribute value size.
+	AttributeValueLengthKey = "OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT"
+
+	// AttributeCountKey
+	// Maximum allowed span attribute count
+	AttributeCountKey = "OTEL_ATTRIBUTE_COUNT_LIMIT"
+
+	// SpanAttributeValueLengthKey
+	// Maximum allowed attribute value size for a span.
 	SpanAttributeValueLengthKey = "OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT"
 
 	// SpanAttributeCountKey
-	// Maximum allowed span attribute count
+	// Maximum allowed span attribute count for a span.
 	SpanAttributeCountKey = "OTEL_SPAN_ATTRIBUTE_COUNT_LIMIT"
 
 	// SpanEventCountKey
-	// Maximum allowed span event count
+	// Maximum allowed span event count.
 	SpanEventCountKey = "OTEL_SPAN_EVENT_COUNT_LIMIT"
 
 	// SpanEventAttributeCountKey
 	// Maximum allowed attribute per span event count.
 	SpanEventAttributeCountKey = "OTEL_EVENT_ATTRIBUTE_COUNT_LIMIT"
 
 	// SpanLinkCountKey
-	// Maximum allowed span link count
+	// Maximum allowed span link count.
 	SpanLinkCountKey = "OTEL_SPAN_LINK_COUNT_LIMIT"
 
 	// SpanLinkAttributeCountKey
-	// Maximum allowed attribute per span link count
+	// Maximum allowed attribute per span link count.
 	SpanLinkAttributeCountKey = "OTEL_LINK_ATTRIBUTE_COUNT_LIMIT"
 )
 
+// firstInt returns the first set int value for an environment variable with
```

Comment:
Take it with a large grain of salt (this is my first review on this project!), but I might consider rephrasing to something like:

> firstInt iterates over each key and checks for an environment variables with the key name; the function returns the first one that can be cast to an int. If no integer is found (or if a non-integer is stored in a key), the function returns defaultValue.

Reply:
Updated.